lib/services.identity.js

```diff
@@ -376,6 +432,60 @@ bedrock.events.on('bedrock-express.configure.routes', function(app) {
       404: 'The identity was not found on the system.'
     }
   });
+
+  app.delete(idPath, ensureAuthenticated, function(req, res, next) {
```

Comment:
I'm not sure what else that impacts right now.  If an id is just marked deleted, do the admin queries still show it?  Can an admin undelete it?  Can a id owner delete themselves?  Do all the other identity queries handle that an id could be marked deleted?